Jenkinsfile

```diff
@@ -255,7 +255,7 @@ export PLAT_CONFIGURE_ARGS="-d $PLAT_CONFIGURE_ARGS"
                     node {
```

Comment:
A couple lines above, you need to rename the stage to `gcc14` as well (line 253).

Reply:
fixed.